.github/workflows/e2e-tests-main-devnet.yml

```diff
@@ -54,6 +54,42 @@ jobs:
           retention-days: 7
 
 
+  build-cliain-image:
+    name: Build docker image for cliain
+    runs-on: ubuntu-20.04
+    steps:
+      - name: GIT | Checkout source code
+        uses: actions/checkout@v2
+
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+
+      - name: Restore cache
```

Comment:
and probably you want to run also
```
      - name: Cleanup cache
        uses: ./.github/actions/post-cache
```
at the end of the job

Reply:
Pretty inconvenient design.

Reply:
:man_shrugging: